<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PointCodes</class>
 <widget class="QDialog" name="PointCodes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>541</width>
     <height>441</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QComboBox" name="cmbModel">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Код:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="txtCode"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Макет:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QPushButton" name="cmdAdd">
          <property name="text">
           <string>Добавить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cmdEdit">
          <property name="text">
           <string>Изменить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cmdRemove">
          <property name="text">
           <string>Удалить</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <widget class="QTableWidget" name="twCodes">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="horizontalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>20</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>20</number>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cmdAdd</sender>
   <signal>clicked()</signal>
   <receiver>PointCodes</receiver>
   <slot>onAddClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>500</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>762</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdEdit</sender>
   <signal>clicked()</signal>
   <receiver>PointCodes</receiver>
   <slot>onEditClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>520</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>752</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdRemove</sender>
   <signal>clicked()</signal>
   <receiver>PointCodes</receiver>
   <slot>onRemoveClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>538</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>108</x>
     <y>781</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>twCodes</sender>
   <signal>itemClicked(QTableWidgetItem*)</signal>
   <receiver>PointCodes</receiver>
   <slot>onElementSelect(QTableWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>93</x>
     <y>800</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onEditClick()</slot>
  <slot>onAddClick()</slot>
  <slot>onRemoveClick()</slot>
  <slot>onElementSelect(QTableWidgetItem*)</slot>
 </slots>
</ui>
